documentation/book/con-reassignment-throttles.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// assembly-scaling-clusters.adoc
+
+[id='con-reassignment-throttles-{context}']
+
+= Reassignment throttles
+
+Reassigning partitions can be a slow process because it can require moving lots of data between brokers.
```

Comment:
Partition reassignment can be a slow process because it involves transferring large amounts of data between brokers.